config.example.ini

```diff
@@ -3,4 +3,7 @@ Token =
 ClientID = 
 OwnerID = 
 [Roles]
-Channel =
\ No newline at end of file
+Channel =
+[Trivia]
+Channel_1 =
+Channel_2 =
```

Comment:
Having 2 channel options is kinda messy. I don't think INI files support arrays, However, it would be great if that could be a list of some sort. I may need to start using JSON/YAML for the config file.